src/node_v8.h

```diff
@@ -34,6 +36,40 @@ class BindingData : public SnapshotableObject {
   SET_MEMORY_INFO_NAME(BindingData)
 };
 
+class GCProfiler : public BaseObject {
+ public:
+  enum class GCProfilerState { kInitialized, kStarted, kStopped };
+  GCProfiler(Environment* env, v8::Local<v8::Object> object);
+  inline ~GCProfiler() override;
+  static void New(const v8::FunctionCallbackInfo<v8::Value>& args);
+  static void StartGCProfile(const v8::FunctionCallbackInfo<v8::Value>& args);
+  static void StopGCProfile(const v8::FunctionCallbackInfo<v8::Value>& args);
+
+  GCProfilerState state();
+  void set_state(GCProfilerState state);
+
+  u_int64_t start_time();
+  void set_start_time(u_int64_t time);
```

Comment:
This looked unnecessary to me. Why don't we make start_time and current_gc_type public, if we allow setter and getter to consume/return data without any changes?

Reply:
Thanks, i have moved `start_time`, `current_gc_type` and `state` to `public`.